README.md

```diff
@@ -12,12 +12,12 @@
 3. ~~optimize the boat velocity with the 3-DOF equlibrium as constraints (Lagrange multipliers)~~
 4. Add all the windage contributions (mast, crew, rigging, etc.)
 5. Optional Delft hydro model
-6. Add dagerboards to the possible appendages  
+6. Add dagger boards to the possible appendages  
 6. ~~tidy plotting and results~~
 
 ### Prerequisites
 
-There are only a few prerequisites to run this code, most python instalation will have them. We advise to use the `environment.yml` file provided in this repo to set-up a new conda environment to use the code (this keeps you machine nice and tidy).
+There are only a few prerequisites to run this code, most python installations will have them. We advise to use the `environment.yml` file provided in this repo to set-up a new conda environment to use the code (this keeps you machine nice and tidy).
```

Comment:
Do we still have the `environment.yml` or is it gone? If so can you change to the `requirements.txt`?

Reply:
I think I deleted it - have updated to reflect requirements plus I made a few other changes to it